crates/solvers/src/infra/config/dex/mod.rs

```diff
@@ -0,0 +1,10 @@
+pub mod balancer;
+mod file;
+pub mod zeroex;
+
+use crate::domain::{dex::slippage, eth};
+
+pub struct BaseConfig {
```

Comment:
Mostly an organizational/naming nit, but to me this is not a "base configuration" but the "dex-solver configuration" (i.e. the configuration options required by the `solver::Dex` and unrelated to the actual DEX API being used).

Maybe it makes sense to have:
```
infra::config::dex
├── solver
├── balancer
└── zeroex
```

Reply:
Makes sense. I think that the new way you organized the configs makes a lot of sense.